src/store/activity/reducers.js

```diff
@@ -176,6 +188,48 @@ const activityReducer = (state = initialState, action) => {
           },
         },
       });
+    case ActionTypes.ACTIVITIES_MUTUAL_LOAD_MORE:
+      return update(state, {
+        mutualActivities: {
+          isError: { $set: false },
+          isLoadingMore: { $set: true },
+        },
+      });
+    case ActionTypes.ACTIVITIES_MUTUAL_LOAD_MORE_SUCCESS: {
+      // Nothing more to add ..
+      if (action.meta.activities.length === 0) {
```

Comment:
try replacing the condition with
 `action.meta.activities.length < action.meta.offset` 
 or similar to have the load more button disappear when less than than 10 are fetched
 Maybe this is not offset but.. page size? This constant should exist somewhere for the call to the core, use the same constant

Reply:
yeah I have seen it but decided to leave it as it is sot it is consistence with activities.
But since you commented I decided to than repair it over there as well.

So I add comment to this task to change it's scope:
https://github.com/CirclesUBI/circles-myxogastria/issues/272

It is not solved now 100% but it improved a lot..